python/pyspark/sql/dataframe.py

```diff
@@ -1256,18 +1256,33 @@ def approxQuantile(self, col, probabilities, relativeError):
         Space-efficient Online Computation of Quantile Summaries]]
         by Greenwald and Khanna.
 
-        :param col: the name of the numerical column
+        :param col: the name of the numerical column, or a list/tuple of
```

Comment:
I'd prefer to incorporate expected types here, see @viirya's doc [here](https://github.com/apache/spark/pull/12207/files#diff-6fc344560230bf0ef711bb9b5573f1faR1184)